src/EnergyPlus/Construction.cc

```diff
@@ -1957,7 +1957,7 @@ bool ConstructionProps::isGlazingConstruction(EnergyPlusData &state) const
     // PURPOSE OF THIS SUBROUTINE:
     // Commonly used routine in several places in EnergyPlus which examines if current
     // construction is glazing construction
-    auto const MaterialGroup = state.dataMaterial->Material(LayerPoint(1)).Group;
+    auto const &MaterialGroup = state.dataMaterial->Material(LayerPoint(1)).Group;
```

Comment:
`MaterialGroup` is a scalar, so no `&` and no `auto`. `DataHeatBalance::MaterialGroup`.